core/templates/dev/head/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -69,6 +70,11 @@ oppia.factory('threadDataService', [
         }
       }
     });
+    var payload = {
+      exploration_id: _expId,
+      thread_id: threadId
+    };
+    $http.post(_FEEDBACK_THREAD_VIEW_EVENT_URL, payload);
```

Comment:
Or, actually -- if this is where we group all the HTTP calls, then a better design approach is to create a new function in the public API for this service that is something like markThreadAsSeen(..., ...). That function then makes the $http POST to the backend. The controller calls this function explicitly when the user clicks the button to open a thread.